packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLogo/__tests__/AppHeaderLogo-test.tsx

```diff
@@ -18,7 +19,7 @@ const renderAppHeaderLogo = (overrideProps?: Partial<AppHeaderLogoItem>) => {
       trackingTarget: 'tracking target',
       ...overrideProps,
     },
-    onClick: jest.fn(),
+    onClick: onClick,
```

Comment:
Very teeny nonblocking nit: we'll lint for this eventually, but just to show off cool JS features, this is equivalent:

```suggestion
    onClick,
```

Reply:
Updated in all the test files - thanks!